lib/openhab/dsl/rules/builder.rb

```diff
@@ -1366,23 +1439,15 @@ def watch(path, glob: "*", for: %i[created deleted modified], attach: nil)
 
         # @!endgroup
 
+        # @!visibility private
         #
-        # Checks if this rule should run on start
-        #
-        # @return [true, false] True if rule should run on start, false otherwise.
-        #
-        def on_start?
-          @on_start.enabled
-        end
-
+        # Run the supplied block inside the object instance of the object that created the rule
         #
-        # Get the optional start attachment
+        # @yield [] Block executed in context of the object creating the rule
         #
-        # @return [Object] optional user provided attachment to the on_start method
         #
-        # @!visibility private
-        def start_attachment
-          @on_start.attach
+        def my(&block)
```

Comment:
I'm guessing this method came back due to an improper merge conflict resolution? it's never used, and can be removed

Reply:
Oops! Yes, I don't remember writing it so I thought I should leave it there. I'll remove it.